sdk/cosmos/.azure-pipelines/cosmos.test.nightly-emulator.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
```

Comment:
While this location is fine for this PR we are working on a new organization of the pipeline files but that we haven't gotten to the python repo yet and we can move these at the same time we organize the others in this repo

Reply:
OK good to know, I thought this was the new organization (instead of just a single top-level set of files)

Reply:
This organization was so last week :)

We are still working on the exact organization and this was one of the incarnations, so you were correct for a point in time. However don't worry about that part as we will move it as necessary.